rpc_proxy.go

```diff
@@ -183,6 +186,10 @@ type rpcProxy struct {
 // bakeSuperMac can be used to bake a new super macaroon.
 type bakeSuperMac func(ctx context.Context, rootKeyID uint32) (string, error)
 
+// lndBasicClient can be used to obtain access to an lnrpc.LightningClient if
+// it is available.
+type lndBasicClient func() (lnrpc.LightningClient, error)
```

Comment:
nit: To make it more obvious this is a function type, should it be called `lndBasicClientFn` or `lndBasicClientGetter` or something like that?